php/media/class-gallery.php

```diff
@@ -153,18 +159,18 @@ public function enqueue_gallery_library() {
 			self::GALLERY_LIBRARY_URL,
 			array(),
 			$this->media->plugin->version,
-			true
+			$this->gallery_in_footer
 		);
 
 		$json_config = wp_json_encode( $this->get_config() );
-		wp_add_inline_script( self::GALLERY_LIBRARY_HANDLE, "var cloudinaryGalleryConfig = JSON.parse( '{$json_config}' );" );
+		wp_add_inline_script( self::GALLERY_LIBRARY_HANDLE, "var CLD_GALLERY_CONFIG = JSON.parse( '{$json_config}' );" );
```

Comment:
`wp_add_inline_script()` puts this JS _after_ the parent JS by default per https://developer.wordpress.org/reference/functions/wp_add_inline_script/ -- is that correct considering we need the widget defaults before we can render the widgets?

Reply:
Found a clean workaround for this!